.github/ISSUE_TEMPLATE/feature-template.md

```diff
@@ -9,27 +9,10 @@ assignees: ''
 ---
 
 ## What is the problem we are solving 
-<!-- Describe the problem this feature is supposed to solve. Should be described in the icebox item (in Discourse). -->
+<!-- Describe the problem this feature is supposed to solve. Should be described in the icebox item. -->
 
 ## Success factors = expected outcome 
 <!-- Describe what is the expected outcome: when the feature is released, what would it look like? What will make you say "the problem is solved"? Can be metrics like "Platform has less than 1% service interruption", or yes/no statements like "People can checkout on mobiles. -->
 
 ## Useful information for inception
 <!-- List here any information that can be useful for the inception stage. -->
-
-## Link to the "Product Development - Backlog" item in Discourse
-<!-- Put the link here, and put this epic link in the Discourse item as well for cross-referencing. -->
- Add a custom footer
- Pages 70
-Home
-Development environment setup
-
-macOS (Sierra, HighSierra and Mojave)
-OS X (El Capitan)
-OS X (Mavericks)
-Ubuntu
-On Heroku
-Rubocop
-General guidelines
-
-Spree Commerce customisation
```

Comment:
I think we can remove this but I am not 100%, @mkllnk can you confirm ?

Reply:
maybe line 20 should be removed only

Reply:
Actually this isn't related to the Discourse SSO integration. 
We still use a Discourse forum for discussing new features (https://community.openfoodnetwork.org/), so this file shouldn't be changed at all.
(Well, actually it could be explained a lot better in this file, but I don't have  suggestion for that)

Reply:
thanks for review! addressed